### DISCUSSION TITLE
Error in fund_and_withdraw.py when running on rinkeby

### DISCUSSION THREAD

**Discussion by BigDog1400** [smartcontractkit/full-blockchain-solidity-course-py#172] [Q&A] [answered]
Hi, i am currently in the lesson 6 of the course. I have already created the fund() function that call the getEntranceFee() method of the smart contract to return entrance_fee needed, and this function runs correctly when it is running in the local blockchain ( just as Patrick does in the video course)
```
def fund():
    fund_me = FundMe[-1]
    account = get_account()
    entrance_fee = fund_me.getEntranceFee() 
    print(f"The current entry fee is {entrance_fee}")
    fund_me.fund({"from": account, "value": entrance_fee})
```

   But, i tried to deploy this contract on rinkeby (success), and run the fund_and_withdraw.py in rinkeby, but, i get the following error:
   
![image](https://user-images.githubusercontent.com/63280663/136702674-6bcce700-0ff2-4514-8bc2-e31adf9fb23f.png)
Why is this happening? I am sending the amount needed, and the FunMe.sol contract has the validation for this values being equal or greater, it should run without problems. In the same lesson when he writes de test for this function, he adds  + 100 to the resul of getEntranceFee(), and if i do the same in the fund(), it works perfectly, but i don't understand why we need to do this, can someone explain this a little bit?

**Comment by PatrickAlphaC** [comment, marked as answer]
Glad that works for you! So the entrance fee could change based on when you call `getEntranceFee`, so you add 100 just to always make sure you have enough.
